js/main.js

```diff
@@ -88,12 +116,31 @@ function handleContentLoad(event) {
 }
 
 function newEntryClick(event) {
-  switchView('entry-form');
+  switchView('entry-form', 'new');
 }
 
-function switchView(view) {
+function switchView(view, form = null) {
   for (var i = 0; i < $viewList.length; i++) {
     if ($viewList[i].getAttribute('data-view') === view) $viewList[i].classList.remove('hidden');
     else $viewList[i].classList.add('hidden');
   }
+  var $pageTitle = document.querySelector('.form-page-title');
+  if (form === 'edit') $pageTitle.textContent = 'Edit Entry';
+  else if (form === 'new') $pageTitle.textContent = 'New Entry';
+}
+
+function clickOnEdit(event) {
+  if (event.target.className !== 'edit-icon') {
+    return;
+  }
+  switchView('entry-form', 'edit');
+  var listID = parseInt(event.target.closest('li').getAttribute('id').slice(2));
+  for (var i = 0; i < data.entries.length; i++) {
+    if (data.entries[i].entryID === listID) data.editing = data.entries[i];
+  }
+  $newEntryForm.elements.title.value = data.editing.title;
+  $newEntryForm.elements.image.value = data.editing.image;
+  $newEntryForm.elements.notes.value = data.editing.notes;
+  var $newPhoto = document.querySelector('.new-entry-img');
```

Comment:
You've queried for this element in two separate functions now, it would be good to change this to a global variable, so you don't need to query every time you use it.